src/chart/components/y_axis/y-axis-scale.handler.ts

```diff
@@ -78,10 +80,21 @@ export class YAxisScaleHandler extends ChartBaseElement {
 			);
 			this.addRxSubscription(
 				this.canvasInputListener.observeDbTap(this.hitTest).subscribe(() => {
+					// apply dbl tap only if single finger taps are made
+					if (this.touches && this.touches?.length > 1) {
+						this.touches = undefined;
+						return;
+					}
+
 					this.dblTapCallback();
 					this.bus.fireDraw();
 				}),
 			);
+			this.addRxSubscription(
+				this.canvasInputListener.observeTouchStart(this.hitTest).subscribe(e => {
```

Comment:
Are u sure that `e.touches` will work correctly with `observeTouchStart `? Because, there is a case when several touches (3 finges ex.) can touch chart one + one (1 finger => 2 fingers => 3 fingers) with some delay asyncronusly. Maybe it is better to use `observeTouchMove `to catch `touches `correctly?

Reply:
This event only triggers when y axis is touched (added hit test in `observeTouchStart`) and touch events work correct if there is a one finger at the axis and then the second one makes touchStart, it will set as 2 touches and handle double tap later correctly